content/docs/api-reference/params_show.md

```diff
@@ -0,0 +1,198 @@
+# dvc.api.params_show()
+
+Load <abbr>parameters</abbr> (name and values) tracked in a <abbr>DVC
+project</abbr>.
+
+```py
+def params_show(
+    *targets: str,  # Optional
+    stages: Optional[Union[str, Iterable[str]]] = None,
+    repo: Optional[str] = None,
+    rev: Optional[str] = None,
+    deps: bool = False,
+) -> Dict:
+```
+
+## Usage:
+
+```py
+import dvc.api
+
+params = dvc.api.params_show()
+```
+
+## Description
+
+Retrieves <abbr>params</abbr> keys and values from a <abbr>DVC project</abbr>
+and returns a dictionary, such as:
+
+```json
+{
+  "split": 0.2,
+  "seed": 20170428
+}
+```
+
+Without arguments, this function will retrieve all params from all tracked
+parameter files (used in any `dvc.yaml` file). This applies to the current
+project version when using Git (including any changes in the working tree).
+
+The function parameters (below) let you restrict what's retrieved.
+
+## Parameters
+
+- `*targets` - paths to one or more valid parameter file(s) to retrieve params
+  from. For example, `"params.py, myparams.toml"`. If no `targets` are provided,
+  all param files tracked in any `dvc.yaml` will be used by default. Note that
+  explicit targets don't have to be used in a `dvc.yaml` (unless `deps=True`).
+
+- `repo` - specifies the location of the DVC project. It can be a URL or a file
+  system path. Both HTTP and SSH protocols are supported for online Git repos
+  (e.g. `[user@]server:project.git`). _Default_: The current project (found by
+  walking up from the current working directory tree).
+
+- `stages` - one or more names of the stage(s) to retrieve params from.
+  _Default_: `None` (all parameters from all stages will be retrieved).
+
+- `rev` - Git commit (any [revision](https://git-scm.com/docs/revisions) such as
+  a branch or tag name, a commit hash or an
+  [experiment](/doc/command-reference/exp) name). If `repo` is not a Git repo,
+  this option is ignored. _Default_: `None` (current working tree will be used)
+
+- `deps` - whether to retrieve only params that are stage dependencies. Accepts
+  `True` or `False` (_default_).
+
+## Example: Filter by stage name(s)
+
+> Working on https://github.com/iterative/example-get-started, file
+> `src/featurization.py`.
```

Comment:
I think we are better off without referencing an example repo because:

1. Users probably aren't familiar with the repo like we are and shouldn't have to check out the repo to understand the context.
2. We probably should update the example repo with `dvc.api.params_show()`, and then we will have to update this ref again.

Reply:
To clarify, all the example content underneath looks good. I just don't think we should reference that it's based on example-get-started.

Reply:
> all the example content underneath looks good

But rn the example uses a git-diff style so it's supposed to be a change based on something:

![image](https://user-images.githubusercontent.com/1477535/176541503-8ceeb00b-3f53-450e-8c46-4d04b744de09.png)

@daavoo mind rewriting this part so no reference is needed yet the usage of the function is explained in the code snippet? Prob no need for git-diff block after all.